CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ set(PROJECT_VERSION "0.4.20")
 project(solidity VERSION ${PROJECT_VERSION})
 
 option(SOLC_LINK_STATIC "Link solc executable statically on supported platforms" OFF)
+option(LLLC_LINK_STATIC "Link lllc executable statically on supported platforms" OFF)
+option(INSTALL_LLLC "Include lllc executable in installation" ON)
```

Comment:
How about making this off by default?

Reply:
After having read up the earlier comments, the benefits are:
- it definitely won't alter the ppa process
- it will not give the impression that `lllc` is a fully maintained, supported and integral part of `solc` (as it is just something allowed to exists here)

Reply:
I'm fine either way. Most people don't build from source anyway.

Reply:
Let's just get this merged today so we can create a release tomorrow.